From d9458f7a82b916efcc7bebf576388ef9bf2e8e45 Mon Sep 17 00:00:00 2001
From: danigm <danigm@wadobo.com>
Date: Wed, 05 Jan 2011 12:57:16 +0000
Subject: Fixed minor bug, it returned NULL when expecting gboolean.

---
diff --git a/backend/pdf/ev-poppler.cc b/backend/pdf/ev-poppler.cc
index ced3ef7..8c5b203 100644
--- a/backend/pdf/ev-poppler.cc
+++ b/backend/pdf/ev-poppler.cc
@@ -2071,7 +2071,7 @@ pdf_document_text_get_text_layout (EvDocumentText  *selection,
 {
 	PopplerPage *poppler_page;
 
-	g_return_val_if_fail (POPPLER_IS_PAGE (page->backend_page), NULL);
+	g_return_val_if_fail (POPPLER_IS_PAGE (page->backend_page), FALSE);
 
 	poppler_page = POPPLER_PAGE (page->backend_page);
 
--
cgit v0.8.3.1
